src/Parser/EnhancementShaman/Modules/ShamanCore/Pandemic.js

```diff
@@ -0,0 +1,54 @@
+import SPELLS from 'common/SPELLS';
+
+import Module from 'Parser/Core/Module';
+
+const PANDEMIC_THRESHOLD = 11500;
```

Comment:
If I recall correctly, the duration of Flametongue is 16s, so the pandemic would be after 11.2s (11200) right?

Reply:
The rule is don't refresh with more than 4.5 seconds left on the debuff. 
Timestamp is record in ms, so entire buff is 16,000 ms, subtract 4.5 s (4500ms), and you get 11,500ms.